pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -543,8 +543,8 @@ func (m *managerImpl) containerEphemeralStorageLimitEviction(podStats statsapi.P
 }
 
 func (m *managerImpl) evictPod(pod *v1.Pod, gracePeriodOverride int64, evictMsg string, annotations map[string]string) bool {
-	// If the pod is marked as critical and static, and support for critical pod annotations is enabled,
-	// do not evict such pods. Static pods are not re-admitted after evictions.
+	// If the pod is marked as critical or static, do not evict such pods.
```

Comment:
At a higher level, I'm wondering if this comment is doing us more harm than good. I worry that it will become out of date and confusing if the implementation of `IsCriticalPod` changes. Thoughts on deleting the `If the pod is marked...` part entirely (and still leaving the `Static pods ...` part)? If someone looking at the code is interested, they can read the `IsCriticalPod` function.